src/migrations.js

```diff
@@ -24,7 +21,7 @@ require.scopes.migrations = (function() {
 var exports = {};
 exports.Migrations= {
   changePrivacySettings: function() {
-    if (!chrome.extension.inIncognitoContext) {
+    if (!chrome.extension.inIncognitoContext && chrome.privacy ) {
```

Comment:
The issue here is that firefox does not have a mode like Chrome's split. Background pages are never run with `inIncognitoContext == true`.

I'm having a hard time debugging this in firefox. I can't figure out how to get a console, or log messages in context where `inIncognitoContext == true`.